native/package.json

```diff
@@ -56,9 +56,9 @@
     "events": "^3.2.0",
     "hashids": "^2.2.8",
     "htmlparser2": "^6.0.0",
-    "i18next": "^19.1.0",
+    "i18next": "19.9.1",
```

Comment:
Why did you remove `^`?

Reply:
Same for most of the other dependencies

Reply:
I just recognized you did the same for babel dependencies in a previous PR, right?

Reply:
I thing it doesn't make much of a difference. With the ^ yarn upgrade will only do minor version upgrades.
But with the lockfile it won't do anything automatically anyway. So if we upgrade we should try to upgrade to the latest version because testing is necessary either way.

The standard behavior of installing a new packet via yarn upgrade is adding the "^" by default. 
I can also re-add it, I don't care